squirrel.worker/src/main/java/org/dice_research/squirrel/encoder/TripleEncoder.java

```diff
@@ -0,0 +1,102 @@
+package org.dice_research.squirrel.encoder;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URISyntaxException;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+import org.apache.jena.graph.Triple;
+import org.apache.jena.util.URIref;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class that can encode triples to UTF8
+ * 
+ * @author Geraldo de Souza Junior gsjunior@mail.uni-paderborn.de
+ *
+ */
+public class TripleEncoder {
```

Comment:
please add more javadoc comments to document the method of the class.

Reply:
done